.gitlab-ci.yml

```diff
@@ -1,9 +1,37 @@
-job:
+slurm-job:
   tags:
     - quartz
-    - shell
+    - batch
+  variables:
+     LLNL_SLURM_SCHEDULER_PARAMETERS: "-N 3 -p pbatch"
   script:
-    - module load cmake/3.9.2
-    - cmake -DWITH_KVTREE_PREFIX=$HOME/kvtree/KVTree.git/install -DWITH_SPATH_PREFIX=$HOME/spath/install -DWITH_AXL_PREFIX=$HOME/axl/AXL.git/install -DMPI=ON
-    - make VERBOSE=1
+    - ./buildme_deps
+    - source /etc/profile.d/z00_lmod.sh
+    - set -x
+    - installdir=`pwd`/install
+    - rm -rf build
+    - mkdir build
+    - cd build
+    - cmake -DWITH_AXL_PREFIX=../install -DCMAKE_INSTALL_PREFIX=$installdir -DVELOC_RESOURCE_MANAGER=SLURM -DCMAKE_BUILD_TYPE=Debug  -DOPENSSL_ROOT_DIR=/usr -DCMAKE_VERBOSE_BUILD=1 ..
+    - make
+    - make install
     - make check
+lsf-job:
+  tags:
+    - lassen
+    - batch
+  variables:
+     LLNL_LSF_SCHEDULER_PARAMETERS: "-nnodes 3 -q pbatch"
+  script:
+    - ./buildme_deps
```

Comment:
Inside the `buildme_deps` script, you do a `module load gcc/8.1.0`. I don't recall if this will persist after running the script, or if you'll end up building filo with the default compiler rather than the same one the dependencies were built with. Do you recall what happens here?